.pre-commit-config.yaml

```diff
@@ -3,10 +3,10 @@ repos:
     rev: stable
     hooks:
       - id: black
-        language_version: python3.7
+        language_version: python3
```

Comment:
I otherwise ran into troubles running pre-commit (actually installing the envs) from my py3.8 dev environment